### DISCUSSION TITLE
how to explain Kidney volume between PK-Sim and article ？

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#857] [Questions & Problems] [answered]
Dear all,
I found a paper which should be published by PK-Sim team that normalize total renal volume with respect to the subject's body surface area。But if I set a person whose information is  European race ,30 years old, male, height is 176cm in PK-sim, the total kidney volume is 0.44 l, which is very different from the results of this article, how to explain this situation?
![image](https://user-images.githubusercontent.com/63133525/102759108-a49c2880-43ae-11eb-9402-8d471e325459.png)

**Comment by prvmalik** [comment]
This paper is not by the PK-Sim team nor reviewed by them. It is by me and my team.

The PK-Sim kidney volume includes the total volume of two kidneys including their blood volume, arteries, veins, glands and all fluids, etc. Stephan Willmann or Jan Schlender may be able to tell you more about this measurement. I merely take it as the healthy reference point.

In the graph above, this value (440 mL/two kidneys) has been normalized to the ultrasound-measured adult reference value for one kidney (150 mL). This is why the y-axis is the 'PK-Sim normalized Kidney volume.'

The purpose of the graph is to understand the fractional change in kidney volume, rather than the absolute value. These fractions are applied as a 'fraction of healthy value' when implementing kidney disease.

There is no normalization by body surface area here.

**Comment by Wenjing1018** [comment]
@prvmalik，
Thank you for your answer,but I think I don't know what is your meaning well. if a person's GFR is 50 ml/min/1.73m2, can I use this formula to calculate kidney volume? Or do I need to make additional changes to this?
Yours,
Sincerely！

**Comment by prvmalik** [comment, marked as answer]
The paper explains how to use the formula. You would take the person's automatically generated healthy kidney volume (Healthy KV) and healthy GFR (GFR_o) from PK-Sim. First normalize the PK-Sim value for GFR (mL/min) to 1.73m2 BSA (I know we have talked about this before).

To get the CKD kidney volume (Diseased KV) for a desired CKD GFR (GFR_t):
![Forum post](https://user-images.githubusercontent.com/29066045/102880738-cd0d3c80-4419-11eb-91e7-60ca4504268f.JPG)

You can see from this formula that the equations on the graph are used to calculate the fraction of KV that should remain 'healthy' in someone who has kidney disease as a function of their GFR vs. their expected healthy GFR for a person of their age.

The fraction represents the theoretical 'chronic kidney disease burden.' Using this method allows you to apply CKD to any individual and conserves the information generated previously for healthy individuals and populations in PK-Sim (Willmann, Schlender, others in various papers).

**Comment by Wenjing1018** [comment]
@prvmalik，
 I understand it well now, thank you for your patient answer, please forgive me for my mistakes I just made.Merry Christmas for you in advance！
Yours,
Sincerely!
